src/App.css

```diff
@@ -1,39 +1,82 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.App {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 3rem 0;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+header h2 {
+  text-align: center;
 }
 
```

Comment:
Your design (It is too small and does not match the given template) 🛠

![image](https://github.com/supreeti/Math-magician-App/assets/78251927/7a062a96-0153-4282-8b17-95b4a0a1176e)

As requested by the previous reviewer, please, create a React component that matches this design: 👀 ✔️ 

![image](https://github.com/supreeti/Math-magician-App/assets/78251927/7720f50e-6e2c-4270-a68a-733954ce5d87)

The Calculator.js component should match the given design: 💪